src/StaticReflection/SourceLocator/ParentAttributeSourceLocator.php

```diff
@@ -55,10 +55,10 @@ public function locateIdentifier(Reflector $reflector, Identifier $identifier):
 
             $class->namespacedName = new FullyQualified($identifierName);
             $fakeLocatedSource = new LocatedSource('virtual', null);
-            $classReflector = new ClassReflector($this);
+            $memoizingReflector = new MemoizingReflector($this);
```

Comment:
shouldn't this read like ?

```suggestion
            $memoizingReflector = new MemoizingReflector(new ClassReflector($this));
```